tests_async/unit/requests/test_download.py

```diff
@@ -374,28 +372,23 @@ async def test_consume(self):
     async def test_consume_with_stream(self):
         stream = io.BytesIO()
         chunks = (b"up down ", b"charlie ", b"brown")
-        await self._consume_helper(stream=stream, chunks=chunks)
+        transport = await self._consume_helper(stream=stream, chunks=chunks)
 
         assert stream.getvalue() == b"".join(chunks)
 
-        # TODO() Check mocks.
```

Comment:
can you expand on what this means? This particular comment? It is in a few places

Reply:
This was to check that we are making the required calls with the mock request/response - e.g. response.method.assert_called_once_with(...).